spec/features/user_session_timeout_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+require 'timecop'
+
+describe 'with devise timeoutable enabled' do
+  after { Timecop.return }
+
+  context 'when user has not signed in' do
+    before { visit root_path }
+
+    it_behaves_like 'not signed in'
+  end
+
+  describe 'user sessions timeout' do
+    context 'when user has signed in' do
+      let(:user) { create(:user, :confirmed, :with_organisation) }
+
+      before do
+        sign_in_user user
+        visit root_path
+      end
+
+      context 'and they have been inactive for 60 minutes' do
```

Comment:
Does it make sense to test the negative case? What happens if a user has been inactive for 59 minutes?

I often like to express the "edges" in acceptance tests - 59 minutes they're still logged in, 60 minutes they're logged out - as I find it explains the feature more clearly.

Not sure if it's worth it here, just a thought.